recordedit/recordEdit.app.js

```diff
@@ -332,17 +330,12 @@
                         $rootScope.displayReady = true;
                         // if there is a session, user isn't allowed to create
                     } else if (session) {
-                        var forbiddenMessage = "You are not authorized to Create entities.";
-                        var forbiddenError = new Error(forbiddenMessage);
-
+                        var forbiddenError = new ERMrest.ForbiddenError(messageMap.unauthorizedErrorCode, messageMap.unauthorizedMessage);
                         forbiddenError.code = errorNames.forbidden;
```

Comment:
You should remove these lines that we manually are changing the error code. The `ERMrest.ForbiddenError` constructor should take care of that for us. So we don't need any of the `forbiddenError.code = errorNames.forbidden;` and `notAuthorizedError.code = errorNames.unauthorized;` that are in this file.

Reply:
The requested changes are now done.